runtime/compiler/CMakeLists.txt

```diff
@@ -354,6 +354,8 @@ create_omr_compiler_library(NAME j9jit
 if(OMR_OS_ZOS)
 	# Workaround a compile problem on z/OS by appending "-O" (to override "-O3").
 	set_property(SOURCE "optimizer/VectorAPIExpansion.cpp" APPEND PROPERTY COMPILE_FLAGS "-O")
+	# Workaround a Open XL compile problem on z/OS by appending "-O0" (to override "-O3").
+	set_property(SOURCE "runtime/codertinit.cpp" APPEND PROPERTY COMPILE_FLAGS "-O0")
```

Comment:
Must this use `-O0`, can we keep more optimizations by using `-O1` or `-O2`?

Reply:
It must be `-O0` , as I tried with `-O1` and `-O2` and still got the same error.

Reply:
This should probably be conditional on whether OpenXL is being used (as opposed to xlc).